crates/ruff_cli/src/diagnostics.rs

```diff
@@ -244,8 +246,15 @@ pub(crate) fn lint_path(
             result,
             transformed,
             fixed,
-        }) = lint_fix(path, package, noqa, settings, &source_kind, source_type)
-        {
+        }) = lint_fix(
+            path,
+            package,
+            noqa,
+            unsafe_fixes,
+            settings,
+            &source_kind,
+            source_type,
+        ) {
```

Comment:
I think Konsti's suggestion (which it's totally up to you whether you take) was to do, like:

```
let var = lint_fix(...);
if let Ok(FixerResult { ... }) = var {
  ....
}
```

...to avoid having a long expanded call on the RHS of the `if-let`. Again, completely up to you.

Reply:
👍 I'll do that separately. I want to use `match` instead of `if matches! / else` just outside this too.